### PR TITLE
Update FAQ - Revenue, disclaimer, sectioning

### PR DESCRIPTION
**Pull request by gamarin2**
- Added Disclaimer
- Changed "reward" to "revenue"
- Added Photon as block reward
- Updated commission mechanism
- Improved sectioning

### CODE REVIEW

File: VALIDATORS_FAQ.md [hunk 1]
```diff
@@ -31,7 +36,7 @@ Any participant in the network can signal that they want to become a validator b
 * Validator's name
 * Validator's website (Optional)
 * Validator's description (Optional)
-* Initial commission rate: The commission rate on fees charged to any delegators
+* Initial commission rate: The commission rate on block provisions, block rewards and fees charged to any delegators
```
Comment (rigelrozanski) [inline VALIDATORS_FAQ.md]: "charged to delegators"

**Comment by rigelrozanski** [comment]
@gamarin2 for a more streamlined review process, can you open up a new branch under Cosmos instead of under garamin2 - This will allow myself or others to push commits to the branch directly

File: VALIDATORS_FAQ.md [hunk 2]
```diff
@@ -110,35 +119,43 @@ Even though delegated funds cannot be stolen by their validators, delegators are
 
 The validator that is selected to propose the next block is called proposer. Each proposer is selected deterministically, and the frequency of being chosen is equal to the relative total stake (where total stake = self-bonded stake + delegators stake) of the validator. For example, if the total bonded stake across all validators is 100 Atoms and a validator's total stake is 10 Atoms, then this validator will be chosen 10% of the time as the next proposer.
 
+### Will validators of the Cosmos Hub ever be required to validate other zones in the Cosmos ecosystem?
+
+Yes, they will. If governance decides so, validators of the Cosmos hub might have to validate other zones in the Cosmos ecosystem. For example, validators of the Cosmos hub will also validate the first public Ethermint zone. Of course, they will be compensated for it in the form of block rewards and transaction fees. These rewards will be paid in the native currency of the Ethermint zone, called Photons.
```
Comment (rigelrozanski) [inline VALIDATORS_FAQ.md]: Yes, they will. Initially, validators of the Cosmos hub will also validate the first public Ethermint zone.  If governance decides so, validators of the Cosmos hub may be required to validate additional zones in the Cosmos ecosystem. As the case with the Ethermint Zone, for each additional zone compensation is to be provided in the form of block rewards and transaction fees.

File: VALIDATORS_FAQ.md [hunk 3]
```diff
@@ -110,35 +119,43 @@ Even though delegated funds cannot be stolen by their validators, delegators are
...
+
+## Staking and incentives
+
 ### What is the incentive to stake?
 
-Each member of a validator’s staking pool earns different types of rewards:
-* **Block provisions:** Native tokens of applications run by validators (e.g. Atoms) are inflated to produce block provisions. These provisions exist to incentivize Atom holders to bond their stake, as non-bonded Atom will be diluted over time. 
+Each member of a validator’s staking pool earns different types of revenue:
+* **Block provisions:** Native tokens of applications run by validators (e.g. Atoms on the Cosmos Hub) are inflated to produce block provisions. These provisions exist to incentivize Atom holders to bond their stake, as non-bonded Atom will be diluted over time. 
+* **Block rewards:** Block rewards are paid in Photons. Initial distribution of Photons will be hard spooned from Ethereum. This means Photons will be emitted 1:1 to Ether.
```
Comment (rigelrozanski) [inline VALIDATORS_FAQ.md]: For the Ethermint zone...

File: VALIDATORS_FAQ.md [hunk 4]
```diff
@@ -110,35 +119,43 @@ Even though delegated funds cannot be stolen by their validators, delegators are
...
+* **Block provisions:** Native tokens of applications run by validators (e.g. Atoms on the Cosmos Hub) are inflated to produce block provisions. These provisions exist to incentivize Atom holders to bond their stake, as non-bonded Atom will be diluted over time. 
+* **Block rewards:** Block rewards are paid in Photons. Initial distribution of Photons will be hard spooned from Ethereum. This means Photons will be emitted 1:1 to Ether. 
 * **Transaction fees:** The Cosmos Hub maintains a whitelist of token that are accepted as fee payment. 
 
-These rewards are divided among validators' staking pools according to each validator’s weight. Then, they are divided among delegators in proportion to each delegator’s stake. Note that a commission on fee rewards for delegators is applied by the validator before they are distributed. However, no commission can be applied on block provisions (see below).
+This total revenue is divided among validators' staking pools according to each validator’s weight. Then, it is divided among delegators in proportion to each delegator’s stake. Note that a commission on delegators' revenue is applied by the validator before it is distributed.
```
Comment (rigelrozanski) [inline VALIDATORS_FAQ.md]: Then within each validator the revenue is divided among delegators....

File: VALIDATORS_FAQ.md [hunk 5]
```diff
@@ -110,35 +119,43 @@ Even though delegated funds cannot be stolen by their validators, delegators are
...
-
-Note that validators cannot set commission on block provisions. This is to prevent a continuous shift in distribution from delegators to validators which would increase the concentration of Atoms and ultimately weaken the security of the system.
+Revenue received by a validator’s pool is split between the validator and its delegators. The validator can apply a commission on the part of the revenue that goes to its delegators. This commission is set as a percentage. Each validator is free to set its initial commission, maximum daily commission change rate and maximum commission. The Cosmos Hub enforces the parameter that each validator sets. These parameters can only be defined when initially declaring candidacy.
```
Comment (rigelrozanski) [inline VALIDATORS_FAQ.md]: last sentence: 
These parameters are defined when initially declaring candidacy, and may only be constrained further with time.

**Comment by rigelrozanski** [comment]
@gamarin2 looks good just a few comments for you to address

File: VALIDATORS_FAQ.md (hunk 2 above)
Comment (gamarin2) [reply VALIDATORS_FAQ.md]: ok

File: VALIDATORS_FAQ.md (hunk 1 above)
Comment (gamarin2) [reply VALIDATORS_FAQ.md]: ok

File: VALIDATORS_FAQ.md (hunk 3 above)
Comment (gamarin2) [reply VALIDATORS_FAQ.md]: ?

Comment (gamarin2) [reply VALIDATORS_FAQ.md]: oh I see

File: VALIDATORS_FAQ.md (hunk 4 above)
Comment (gamarin2) [reply VALIDATORS_FAQ.md]: ok

**Pull request merged**